spec/plsql/procedure_spec.rb

```diff
@@ -2251,3 +2251,49 @@ def new_candidate(status)
   end
 
 end
+
+describe '#get_argument_metadata' do
+  before do
+    plsql.execute <<-SQL
+      CREATE OR REPLACE FUNCTION hr.magic_number(p_num INTEGER #{defaulted_value})
```

Comment:
I'm wondering about a better name - `defaulted_value` sounds more like the value itself (21), while we expect the whole expression/clause here.